README.md

```diff
@@ -36,19 +36,18 @@ Bergamot translator provides a unified API for ([Marian NMT](https://marian-nmt.
     cd bergamot-translator
     ```
 
-3. Download files (only required if you want to package files in wasm binary)
+3. Download files (only required if you want to perform inference using build artifacts)
 
-    This step is only required if you want to package files (e.g. models, vocabularies etc.)
-    into wasm binary. If you don't then just skip this step.
+    It packages the vocabulary files into wasm binary, which is required only if you want to perform inference.
```

Comment:
Suggest simplifying the language here. `which is required only if you want to perform inference` to `which is required to improve the translation speed`

Reply:
As per my comment above, this comment in invalid. So, ignoring this one 👍